srcpkgs/ta-lib/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'ta-lib'
+pkgname=ta-lib
+version=0.4.0
+revision=1
+wrksrc="${pkgname}"
+build_style=configure
+configure_args="--prefix=/usr"
+short_desc="Technical Analysis Library for analyzing financial markets trend"
+maintainer="Jean-Philippe Menil <jpmenil@gmail.com>"
+license="custom:TA-Lib BSD License"
+homepage="http://ta-lib.org"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}-src.tar.gz"
+checksum=9ff41efcb1c011a4b4b6dfc91610b06e39b1d7973ed5d4dee55029a0ac4dc651
+patch_args="-Np1"
+
+archs="i686 x86_64 x86_64-musl"
+
+do_build() {
```

Comment:
Default do_build and do_install looks to fit, please move vlicense to post_install if I'm right.

Reply:
seem that config.guess is out of date, i'm finally able to build on aarch64.
Dunno how i will manage that in the template, maybe a case for arm?